### PR TITLE
chore: update bulk insert csv tool to add more normalizing

### PR DESCRIPTION
**Pull request by Goncharo**
refs. metriport/metriport-internal#690

### Dependencies

n/a

### Description

update bulk insert csv tool to add more normalizing

### Release Plan

asap

### CODE REVIEW

File: packages/utils/src/bulk-insert-patients.ts [hunk 1]
```diff
@@ -86,7 +97,19 @@ function normalizeCity(city: string): string {
...
+}
+
+function normalizeZip(zip: string): string {
+  return zip.trim();
+}
+
+function normalizeDate(date: string): string {
+  const trimmedDate = date.trim();
+  if (!dayjs(trimmedDate, "YYYY-MM-DD", true).isValid()) {
+    throw new Error(`Invalid state ${date}`);
```
Comment (leite08) [inline packages/utils/src/bulk-insert-patients.ts:110]: typo, invalid "date"

File: packages/utils/src/bulk-insert-patients.ts [hunk 2]
```diff
@@ -77,6 +78,16 @@ function normalizeName(name: string): string {
   return toTitleCase(name);
 }
 
+function normalizePhone(phone: string): string {
+  if (phone.length === 11 && phone[0] === "1") {
```
Comment (leite08) [inline packages/utils/src/bulk-insert-patients.ts:82]: Maybe `.trim()` before checking for length?

**Review by leite08 — APPROVED** [review]
Left some comments, but LGTM

File: packages/utils/src/bulk-insert-patients.ts (hunk 2 above)
Comment (Goncharo) [reply packages/utils/src/bulk-insert-patients.ts:82]: yeah good point

**Pull request merged**